src/router.ts

```diff
@@ -1372,7 +1541,237 @@ export async function handleSlackVoterThreadMessage(
     `${userId}:${twilioPhoneNumber}`
   )) as UserInfo;
   // Only relay Slack messages from the active Slack thread.
-  if (userInfo.activeChannelId === reqBody.event.channel) {
+  if (
```

Comment:
Can we pull this stuff out into a separate function or file? This function is getting pretty hard to follow -- maybe something like `if (handleCommand(messageToSend)) { return; }` and then `handleCommand()` can return `true` if the message was a command and `false` if it was a normal message.

Reply:
And then that function can help DRY out some of this, like handling the ACL check at the top of each of these branches

Reply:
Yep--updated!